clang/include/clang/Basic/Builtins.td

```diff
@@ -4914,3 +4914,9 @@ def ArithmeticFence : LangBuiltin<"ALL_LANGUAGES"> {
   let Attributes = [CustomTypeChecking, Constexpr];
   let Prototype = "void(...)";
 }
+
+def CountedByRef : Builtin {
+  let Spellings = ["__builtin_counted_by_ref"];
+  let Attributes = [NoThrow, CustomTypeChecking];
+  let Prototype = "int(...)";
```

Comment:
Just to make sure. I guess the prototype doesn't match (`void *` vs `T *`). Is it still fine?

Reply:
For a custom type checking, the prototype is basically meaningless. I'd rather not mess with this as it's not very important.